py3status/modules/arch_updates.py

```diff
@@ -71,10 +71,13 @@ def post_config_hook(self):
             self.include_aur = True
             self.include_pacman = True
 
-        # check cower installed
-        if self.include_aur and not self.py3.check_commands(['cower']):
-            self.py3.notify_user('cower is not installed cannot check aur')
-            self.include_aur = False
+        if self.include_aur:
+            if self.py3.check_commands(['cower']):
+                self._check_aur_updates = self._check_aur_updates_cower
+            elif self.py3.check_commands(['yay']):
+                self._check_aur_updates = self._check_aur_updates_yay
+            else:
+                self.py3.notify_user('cower/yay is not installed cannot check aur')
```

Comment:
@simon04 thanks for the new feature

we still need to set `self.include_aur = False` in no aur helper is available.

Reply:
Thanks for spotting this glitch. I added the line to the `else` branch.